backend/app/models/record/record.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import BaseModel
+
+
+class AbstractInfo(BaseModel):
+    """Pydantic Model that holds the Abstract Information of a Wikipedia Article."""
+
+    title: str = ""
```

Comment:
One or more of these is required?
Seems like these records need more validation.

Use None instead of empty strings for nulls. Blank strings (i.e., strings with only space characters) should not be allowed; can share a validator for that.